pkg/uki/install.go

```diff
@@ -50,10 +56,14 @@ func (i *InstallAction) Run() (err error) {
 		MountPoint: constants.UkiSource,
 	}
 
-	err = e.MountImage(image)
-	if err != nil {
-		return err
+	mounted, err := i.cfg.Mounter.IsMountPoint(constants.UkiCdromSource)
```

Comment:
argh shit, sorry this wont work. The mounter in here is a FakeMounter as we didnt needed a real mounter, soi this check is done in memory mounts only, not real ones....

Reply:
or not, fuck let me check Im a bit confused today

Reply:
yeah Im stupid, this is good, I was confusing this with enki.